src/backend/data/schema.js

```diff
@@ -439,6 +439,12 @@ const GraphQLCallAssignment = new GraphQLObjectType({
         else
           return group.query
       }
+    },
+    listContainer: {
```

Comment:
Hm, not sure that it makes sense to put a listContainer reference here as that's not something that logically makes sense to be on a call assignment.  Instead, can you just pass the fragment to the top-level listcontainer in the react components?